### PR TITLE
oksh: add completion

### PR DESCRIPTION
**Pull request by EmilySeville7cfg**
## Description

Add [`oksh`](https://manned.org/oksh.1) completion.

## TODOs:

- [ ] Changes to fish usage are reflected in user documentation/manpages.
- [ ] Tests have been added for regressions fixed
- [ ] User-visible changes noted in CHANGELOG.rst

### CODE REVIEW

**Comment by faho** [comment]
Okay, can I ask you to, in future, group these very simple completions for very simple (argument-wise) commands together in a single PR to make it easier to review?

**Comment by EmilySeville7cfg** [comment]
> Okay, can I ask you to, in future, group these very simple completions for very simple (argument-wise) commands together in a single PR to make it easier to review?

By all means, you can. :)

File: share/completions/oksh.fish [hunk 1]
```diff
@@ -0,0 +1,7 @@
+set -l options allexport notify noclobber errexit noglob trackall keyword monitor noexec privileged stdin nounset verbose markdirs xtrace
+complete -c oksh -a "+{a,b,C,e,f,h,i,k,l,m,n,p,r,u,v,X,x,o}"
+complete -c oksh -a "-{a,b,C,e,f,h,i,k,l,m,n,p,r,u,v,X,x,o}"
```
Comment (mqudsi) [inline share/completions/oksh.fish:3]: These completions are really opaque to the point of not serving any purpose, and I would be against merging this as-is.

The completions you are specifying here are abbreviated versions of the options available for oksh's `set` builtin. I think they should be presented as short/long arguments with attached descriptions. (See [the man page](https://manned.org/oksh.1).)

It's up to you if you want to implement them or close out the PR.

**Review by mqudsi — CHANGES_REQUESTED** [review]

File: share/completions/oksh.fish (hunk 1 above)
Comment (EmilySeville7cfg) [reply share/completions/oksh.fish]: Fixed. :)

**Comment by faho** [comment]
Merged, thanks.

**Pull request merged**